### ISSUE TITLE
Uncompressed VCF variants

### ISSUE THREAD

**Issue by fellen31** [seppinho/nft-vcf#6]
Hi, 

Thanks for writing this plugin!

I'm having trouble using variants/variantsMD5 on uncompressed VCFs, header seems to work fine.
Is it supported to be supported? 

/Felix

**Comment by seppinho** [comment]
Correct, for some methods, we require a BGZF-compressed file (.vcf.gz) because we are using the query functionality of HTSJDK, which requires an index file. One approach could be to compress the VCF file using BGZIP and generate the index directly within the code. Would it also be possible for you to work directly with a .vcf.gz file?

**Comment by seppinho** [comment]
I understand. I'll do my best to implement this feature. If you feel like contributing to a PR, let me know!
